components/cloud-foundry/frontend/src/view/applications/application/services/service-card/service-card.html

```diff
@@ -1,61 +1,44 @@
-<div class="service-card">
-  <div class="service-icon">
-    <img ng-src="{{ serviceCardCtrl.service.entity.extra.ImageURL }}"
-      default-image default-image-src="/images/services/App_services_72.png"/>
-  </div>
-  <div class="service-block">
-    <div class="service-title">
-      <div class="service-text">
-        <h4 class="font-semi-bold">{{ ::serviceCardCtrl.service.entity.label }}</h4>
-      </div>
-      <div class="service-metadata">
-        <dl>
-          <dt class="font-semi-bold" translate>app.app-info.app-tabs.services.card.version</dt>
-          <dd>{{ ::serviceCardCtrl.service.entity.extra.Version || serviceCardCtrl.service.entity.version || '-'}}</dd>
-        </dl>
-      </div>
-      <div class="service-actions-menu" ng-hide="serviceCardCtrl.hideServiceActions()">
-        <actions-menu actions="serviceCardCtrl.actions" ng-if="!serviceCardCtrl.allowAddOnly && serviceCardCtrl.canBind"
-          action-target="service"
-          menu-position="actions-menu-right">
-        </actions-menu>
+<gallery-card
+  class="cluster-gallery-card"
+  card-actions="serviceCardCtrl.actions"
+  card-data=serviceCardCtrl.cardData
+>
+  <div class="service-catalogue-card">
+    <div class="service-catalogue-card__meta">
+      <div class="service-catalogue-card__meta-split">
+        <div class="service-catalogue-card__meta-left">
```

Comment:
Think service-catalogue-card__meta-left needs some right padding... or right needs some left padding. Squishing the window leads to label and value right next to each other